<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetMatchParamWindow</class>
 <widget class="QDialog" name="SetMatchParamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="Window_Size" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QLabel" name="Window_SizeLabel">
        <property name="text">
         <string>Window Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Window_SizeHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Max_Disparity" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Max_DisparityLabel">
        <property name="text">
         <string>Max Disparity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Max_DisparityHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="PreFilterCarp" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="PreFilterCarpLabel">
        <property name="text">
         <string>Pre Filter Carp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="PreFilterCarpHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="TextureThreshold" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="TextureThresholdLabel">
        <property name="text">
         <string>Texture Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="TextureThresholdHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="UniquenessRatio" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="UniquenessRatioLabel">
        <property name="text">
         <string>Uniqueness Ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="UniquenessRatioHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="SpeckleWindowSize" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="SpeckleWindowSizeLabel">
        <property name="text">
         <string>Speckle Window Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SpeckleWindowSizeHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="SpeckleRange" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="SpeckleRangeLabel">
        <property name="text">
         <string>Speckle Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="SpeckleRangeHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Disp12MaxDiff" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="Disp12MaxDiffLabel">
        <property name="text">
         <string>Disp12 Max Diff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Disp12MaxDiffHorizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Algorithm" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="AlgorithmLabel">
        <property name="text">
         <string>SGBM Algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AlgorithmComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="ControlButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ApplyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
